packages/next/src/server-build.ts

```diff
@@ -676,18 +681,77 @@ export async function serverBuild({
     );
 
     for (const group of combinedGroups) {
+      const groupPageFiles: { [key: string]: PseudoFile } = {};
+
+      for (const page of [...group.pages, ...internalPages]) {
+        const pageFileName = path.normalize(
+          path.relative(baseDir, lambdaPages[page].fsPath)
+        );
+        groupPageFiles[pageFileName] = compressedPages[page];
+      }
+
+      const updatedManifestFiles: { [name: string]: FileBlob } = {};
+
+      if (isCorrectManifests) {
+        // filter dynamic routes to only the included dynamic routes
+        // in this specific serverless function so that we don't
+        // accidentally match a dynamic route while resolving that
+        // is not actually in this specific serverless function
+        for (const manifest of [
+          'routes-manifest.json',
+          'server/pages-manifest.json',
+        ]) {
+          const fsPath = path.join(entryPath, outputDirectory, manifest);
+
+          const relativePath = path.join(path.relative(baseDir, fsPath));
```

Comment:
Is it expected that path.join() has one param instead of 2?

Reply:
Looks like the `join` isn't needed here and must have been left over.